### PR TITLE
Add: Blog post about unbunching feature

### PR DESCRIPTION
**Pull request by 2TallTyler**
A blog post to lead up to the release of OpenTTD 14.

I will also be writing a post about NotDaylength. I tease this at the end of the post, but could replace it with whichever article we decide to run next. I think teasing the next blog post keeps the interest up so I'd love to see a similar mention in the posts by @zephyris and @Kuhnovic, once we determine the order. 😃

### CODE REVIEW

**Comment by zephyris** [comment]
Looks good, but I'm not very good at proofreading for typos/grammar!

I think an image would go a long way, to make it a bit more visually impactful and to give players a pointer. Something like this:
image

And, 100% agree about teasing the next blog post.

**Comment by 2TallTyler** [comment]
I adopted your photo and teased your font blog post as the next one!

**Comment by 2TallTyler** [comment]
April 1 is a Friday, so I think we should make our dev diaries on Fridays, starting this week (9 February). If we're happy with that, I can rename the post for publishing on that date. You just won't be able to see the preview once I do that, so review it now! 😃

File: _posts/2024-02-06-unbunching.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## A new approach
+
+There is one place where vehicles cannot deadlock: Depots.
+
+Vehicles in depots don't exist in the game world, so they cannot block each other.
+This is the trick to the new unbunching feature in OpenTTD 14.
+
+![The depot menu, with its new Unbunch action]({% link /static/img/post_2024-02-06-bus-bunching/unbunch.png %})
+
+Depots gain a new action type marking them as the unbunching point for a vehicle, chosen from the depot action dropdown where you'd choose "Service," "Stop," etc.
```
Comment (zephyris) [inline _posts/2024-02-06-unbunching.md:33]: I suspect some players won't know vehicles can have orders to visit a depot! Also probably need to mention earlier that this works for shared orders. Perhaps "Vehicles can have orders to get serviced or stop at a depot. This feature adds a new type of depot order, marking that depot as the unbunching point for a group of vehicles which share orders. This is chosen from the ...."

File: _posts/2024-02-06-unbunching.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+When a vehicle enters a depot, it records the duration of the trip it just completed.
+When it tries to exit a depot, it checks if its next departure time has arrived yet.
+If we are past the departure time, it leaves and immediately calculates the departure time of the _next_ vehicle.
+
+This calculation is simple: Take the average last trip duration of all vehicles, divide by the number of vehicles to get the interval between trips, then add that interval to the current time.
+Because we look at the average trip duration, inconsistencies due to traffic or routing are canceled out.
+If vehicles are extremely delayed or if new vehicles get added to the route, the interval takes a few cycles to stabilize, but it will work itself out shortly.
+
+A vehicle can only have one depot order marked for unbunching, so if you try and add a second you get an error.
+In addition, unbunching cannot be used with full load or conditional orders.
```
Comment (zephyris) [inline _posts/2024-02-06-unbunching.md:47]: Maybe say why: "because these types of orders can take a very variable amount of time" or similar.

**Review by zephyris — COMMENTED** [review]
A few suggestions, take with a pinch of salt.

You could also include an image of with and without unbunching?
image

**Comment by 2TallTyler** [comment]
Thanks @zephyris! ❤️

**Comment by 2TallTyler** [comment]
This post is now scheduled for tomorrow, Saturday 10 February, 2024.

Steam image:
unbunch-steam

[Unbunch your buses.zip](https://github.com/OpenTTD/website/files/14224509/Unbunch.your.buses.zip)

File: _posts/2024-02-10-unbunching.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+There is one place where vehicles cannot deadlock: Depots.
+
+Vehicles in depots don't take up space on roads or rails in the game world, so they cannot block each other.
+This is the trick to the new unbunching feature in OpenTTD 14.
+
+![The depot menu, with its new Unbunch action]({% link /static/img/post_2024-02-06-bus-bunching/unbunch.png %})
+
+Vehicles can have orders to get serviced or stop at a depot.
+This feature adds a new type of depot order, marking that depot as the unbunching point for a group of vehicles which share they orders.
```
Comment (JGRennison) [inline _posts/2024-02-10-unbunching.md:36]: they orders -> their orders?

File: _posts/2024-02-10-unbunching.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+<!-- more -->
+
+![A comparison of red bunched buses versus purple unbunched buses]({% link /static/img/post_2024-02-06-bus-bunching/bunched-comparison.png %})
+
+To keep vehicles spaced evenly, we can either speed up or slow down vehicles.
+Speeding up vehicles is tricky even in real life, requiring adding extra vehicles, skipping stops, or turning a vehicle around before its final destination.
+It's much easier to simply slow them down. Real transit schedules have extra time built in to pad the schedule, often coinciding with the driver's breaks.
+A late-running vehicle can take a shorter break and depart on time. This is also possible in OpenTTD using timetables, but that's a lot of manual work.
+The most popular fork of OpenTTD, JGR's Patch Pack, includes an "autoseparation" feature that uses timetables automatically to keep vehicles spread out.
```
Comment (JGRennison) [inline _posts/2024-02-10-unbunching.md:20]: that uses timetables automatically to -> that uses timetables to automatically
seems to read better?

**Review by TrueBrain — APPROVED** [review]

**Pull request merged**